src/common/SPELLS/bfa/azeritetraits/shaman.js

```diff
@@ -5,6 +5,53 @@
  */
 
 export default {
+  // Elemental
```

Comment:
I think I prefer this method of putting all azerite related ids inside the azeritetraits folder. Dont think it will block PRs however people decide to do it though.